spec/src/main/asciidoc/appendix.adoc

```diff
@@ -38,6 +38,47 @@ https://github.com/unitsofmeasurement[UoM, JSR 363]
 
 http://metrics20.org/spec/[Metrics 2.0]
 
+
+
+=== Application REST metrics
+
+Metrics gathered from RESTful stats are optional and therefore may not be available in every implementation. These metrics that are created by gathering RESTful stats are to be portable between different implementations and as such are not valid as optional vendor metrics. These metrics belong under the application scope as they are metrics derived from observing the use of RESTful resources in the application.
+
+It is up to the implementation to decide how to enable the REST metrics.
+
+These metrics require that RESTful stats are gathered from all RESTful resources.
+
+
+*(Optional) RESTRequests*
+[cols="1,4"]
+|===
+|Name| rest.request.total
+|DisplayName| Total Requests
+|Type| Counter
+|Unit| None
+|Multi| true
+|Tags| {class=%s1,method=%s2}
+|Description| The number of invocations of this RESTful resource method since the start of the server.
+|Notes|The `%s1` should be substituted with the fully qualified name of the RESTful resource class. 
+  
+The `%s2` should be substituted with the name of the RESTful resource method and appended with its parameter types using an underscore `\_`.  Multiple parameter types are appended one after another (e.g. `<methodName>_<paramType1>_<paramType2>`).
+|===
+
+*(Optional) RESTResponseTotal*
+[cols="1,4"]
+|===
+|Name| rest.elapsed.total
+|DisplayName| Total Response Time
+|Type| Gauge
+|Unit| Nanoseconds
+|Multi| true
+|Tags| {webAppName=%s1,class=%s2,method=%s3}
```

Comment:
Should we have `webAppName` here? I vaguely remember we agreed to not include this, instead we can simply include a note that links to the chapter https://github.com/eclipse/microprofile-metrics/blob/2.2.1/spec/src/main/asciidoc/architecture.adoc#usage-of-microprofile-metrics-in-application-servers-with-multiple-applications and say that the same applies here

Reply:
Ah sorry, that was left behind. I removed it off of the other metric but not this one.